modules/javafx.controls/src/main/java/javafx/scene/control/SplitMenuButton.java

```diff
@@ -85,6 +110,26 @@ public SplitMenuButton() {
      * @param items The items to show within this button's menu
      */
     public SplitMenuButton(MenuItem... items) {
+        this(null, null, items);
+    }
+
+    /**
+     * Creates a new split menu button with the given text and graphic to
+     * display on the button, and inserts the given items
+     * into the {@link #getItems() items} list.
+     *
+     * @param text the text to display on the menu button
+     * @param graphic the graphic to display on the menu button
+     * @param items The items to display in the popup menu.
```

Comment:
Minor: description should start with a small letter and not end with a dot.

Reply:
that was lifted straight from MenuButton...  but you are right of course.